ddl/ddl_api.go

```diff
@@ -3045,9 +3062,10 @@ func (d *ddl) AlterTable(ctx context.Context, sctx sessionctx.Context, ident ast
 	}
 
 	if len(validSpecs) > 1 {
+		continuE := false
```

Comment:
continuE is not a good name.

Reply:
This will be removed finally. In this PR, we only support the multi-schema change for `ADD COLUMN`.

Reply:
I have updated it to "useMultiSchemaChange"